tests/test_continuous.py

```diff
@@ -24,7 +25,7 @@ def test_uniform(lower, upper):
     """Test the continuous uniform distribution."""
     scale = upper-lower
     values = stats.uniform(loc=lower, scale=scale).rvs(100)
-    dist = UniformDistribution.fit(values)
+    dist = UniformDistribution.fit(values, BasicPrivacy())
```

Comment:
Now that I see this; should we add BasicPrivacy as a default parameter value in BaseDistribution? So that you don't always need to do this whenever you're testing distributions?